compute_tools/src/bin/compute_ctl.rs

```diff
@@ -54,9 +55,15 @@ use compute_tools::monitor::launch_monitor;
 use compute_tools::params::*;
 use compute_tools::spec::*;
 
+const BUILD_TAG_DEFAULT: &str = "local";
+
 fn main() -> Result<()> {
     init_tracing_and_logging(DEFAULT_LOG_LEVEL)?;
 
+    let build_tag = std::env::var("BUILD_TAG").unwrap_or(BUILD_TAG_DEFAULT.to_string());
```

Comment:
should this be [`env!`](https://doc.rust-lang.org/std/macro.env.html) instead of `env::var`? 

* `env!` will be expanded at compile time, and the build arg will be part of the binary compiled.
* `env::var` is retrieved at runtime, and therefore if we copy the binary out of the original docker image, this BUILD_TAG will be unset.

Reply:
fixed